### PR TITLE
Make more changes

### PR DESCRIPTION
**Pull request by butteredyakiimo**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -172,7 +173,7 @@ The same process can be applied to other fields.
 
 Entering <code>save</code> saves the changes you have made into storage.
 
-Let's say after changing the name of the fosterer, you want to save your changes.
+Suppose you want to save your changes after changing the name of the fosterer,.
```
Comment (nananakx-x) [inline docs/UserGuide.md]: no comma before full stop here

**Review by nananakx-x — COMMENTED** [review]
lgtm, suggested a minor change

File: docs/UserGuide.md (hunk 1 above)
Comment (butteredyakiimo) [reply docs/UserGuide.md]: oops rectifying now

**Review by nananakx-x — APPROVED** [review]
minor fix to UG

**Pull request merged**